xmlsec1/tests/test.bats

```diff
@@ -0,0 +1,6 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "Version matches" {
+  result="$(hab pkg exec ${TEST_PKG_IDENT} xmlsec1 --version | head -1 | awk '{print $2}')"
+  [ "$result" = "${TEST_PKG_VERSION}" ]
```

Comment:
For new tests I think we are trying to use `diff` for comparison so that we get pretty test output.